### ISSUE TITLE
[BUG] Dropdown - Canvas section - Under accessibility sections, 2 violations is displayed 

### ISSUE THREAD

**Issue by HarshithaNous** [trimble-oss/modus-web-components#1195]
[BUG] Dropdown Web component - Canvas section - Under accessibility sections, 2 violations is displayed  as shown in the below screenshots

1. Go to Dropdown web component
2. Click on Canvas section
3. Under accessibility section, verify if Violations is displayed

**Expected behavior**
Accessibility should display with 0 violations

**Current behavior**
Accessibility is displayed with 1 violation

Please refer the below screenshots:
![image](https://user-images.githubusercontent.com/129380495/232827686-021ba935-e685-4894-9e34-92777e086489.png)

![image](https://user-images.githubusercontent.com/129380495/232828286-7d5083b1-437c-4340-a4f7-33eabd3cfc4e.png)

- Browser [Chrome, Edge and Firefox]
